js/type_directives.ts

```diff
@@ -33,15 +38,24 @@ export function getMappedModuleName(
  */
 const typeDirectiveRegEx = /@deno-types\s*=\s*(["'])((?:(?=(\\?))\3.)*?)\1/gi;
 
-/** Matches `import` or `export from` statements and parses out the value of the
+/** Matches `import` statements and parses out the value of the
+ * module specifier in the second capture group:
+ *
+ *      import "./foo.js"
+ *
+ * [See Diagram](https://bit.ly/2maRCm8)
+ */
+const importRegEx = /(?:import|export)\s+(["'])((?:(?=(\\?))\3.)*?)\1/;
```

Comment:
I think it would be better as a single RegEx.  The following keeps the same capture groups but makes the `from` optional.  While `export "foo";` isn't valid, it is unlikely to ever cause an issue.

```js
const importExportRegEx = /(?:import|export)\s+(?:[\s\S]*?from\s+)?(["'])((?:(?=(\\?))\3.)*?)\1/;
```